test/common.js

```diff
@@ -228,13 +228,24 @@ exports.spawnPwd = function(options) {
 };
 
 exports.platformTimeout = function(ms) {
-  if (process.arch !== 'arm')
-    return ms;
+  var newTimeout = ms;
+  if (process.arch === 'arm') {
+    if (process.config.variables.arm_version === '6') {
+      newTimeout *= 7;  // ARMv6
+    } else {
+      newTimeout *= 2;  // ARMv7 and up.
+    }
+  }
 
-  if (process.config.variables.arm_version === '6')
-    return 7 * ms;  // ARMv6
+  if (process.features.debug) {
+    newTimeout *= 5;
+  }
+
+  if (process.env.TEST_TIMEOUT_MULTIPLIER !== undefined) {
+    newTimeout = ms * process.env.TEST_TIMEOUT_MULTIPLIER;
+  }
```

Comment:
I added for example asan debug builds when the default debug timeout isn't enough.